util/Cargo.toml

```diff
@@ -8,3 +8,11 @@ workspace = ".."
 [dependencies]
 chrono = "0.4.9"
 failure = "0.1.6"
+lazy_static = "1.4.0"
+log = "0.4.8"
+ntp = {git="https://github.com/JeffBelgum/ntp/"}
+time = "0.1.42"
+
+[dependencies.serde]
```

Comment:
I prefer the alternative syntax:

    serde = { version = "1.0.101", features = ["derive"] }